Readme.md

```diff
@@ -46,14 +46,19 @@ with minimal configuration.
 npm install -g aws-cdk
 ```
 
-2. Create a new project directory with the following in a `deps.edn` file.
+2. Create a new project directory with the following in a `deps.edn`
+   file. You will also need to include the maven dependency for any
```

Comment:
s/maven/Maven/.

Also, a link to all the dependencies would be nice, e.g.,

See the [Maven Repository](https://mvnrepository.com/artifact/software.amazon.awscdk) for available JARs.